src/styles/utils/FontUtils/fontFamily/multiFontFamily/index.ts

```diff
@@ -36,7 +36,7 @@ const fontFamily: FontFamilyStyles = {
     },
 
     EXP_NEUE: {
-        fontFamily: 'Expensify Neue, Segoe UI Emoji, Noto Color Emoji',
+        fontFamily: 'Expensify Neue, Segoe UI Emoji, Noto Color Emoji, Custom Emoji Font',
```

Comment:
Why do we need to add this as a backup font here? I would rather not include it as a backup given that it literally only has one character in the font.

Reply:
I think we should just wrap the custom emoji in some kind of class that has the specific font family we need. Can we please try that?

Reply:
Hmm I don't think I know enough about this stuff to know what the pros/cons of adding it as a backup font would be. Curious what Georgia thinks here.